packages/pricing/src/models/price-list-rule-value.ts

```diff
@@ -25,6 +33,25 @@ export default class PriceListRuleValue {
   @Property({ columnType: "text" })
   value: string
 
+  @Property({
+    onCreate: () => new Date(),
+    columnType: "timestamptz",
+    defaultRaw: "now()",
+  })
+  created_at: Date
+
+  @Property({
+    onCreate: () => new Date(),
+    onUpdate: () => new Date(),
+    columnType: "timestamptz",
+    defaultRaw: "now()",
+  })
+  updated_at: Date
+
+  @Index({ name: "IDX_price_list_rule_value_deleted_at" })
```

Comment:
todo: you can use the `createPsqlIndexStatementHelper` instead and create a partial index where deleted at is not null